isis/src/lro/apps/lrowac2pds/lrowac2pds.cpp

```diff
@@ -0,0 +1,485 @@
+/** This is free and unencumbered software released into the public domain.
+
+The authors of ISIS do not claim copyright on the contents of this file.
+For more details about the LICENSE terms and the AUTHORS, you will
+find files of those names at the top level of this repository. **/
+
+/* SPDX-License-Identifier: CC0-1.0 */
+#include "lrowac2pds.h"
+
+#include <cstdio>
+#include <QString>
+#include <QRegExp>
+#include "FileList.h"
+#include "Brick.h"
+#include "OriginalLabel.h"
+#include "Brick.h"
+#include "History.h"
+#include "Stretch.h"
+#include "iTime.h"
+#include "ProcessExport.h"
+#include "PvlToPvlTranslationManager.h"
+#include "PvlFormatPds.h"
+#include "md5.h"
+#include "md5wrapper.h"
+#include "OriginalLabel.h"
+#include <sstream>
```

Comment:
This is padentic and I realize this is how other files are structured as well, but why not place these includes in the header file?

Reply:
That is a good question that I don't have the answer for. I have just been following the instructions in the wiki for writing tests in which we copy everything over from the app's main.cpp and stick to the header [template](https://github.com/USGS-Astrogeology/ISIS3/wiki/Writing-ISIS3-Tests-Using-Gtest-and-Ctest#applicationh-template) provided. So I really am not sure as to the reason why we are doing it this way.

Reply:
Thank you for pointing me towards that documentation. I did not get the chance to work on this project so I was not aware of it! I think it is totally fine the way it is, I was just curious if there was a specific reason for this choice (probably was, but it is lost to time)